tools/pipelines/templates/build-npm-package.yml

```diff
@@ -452,6 +452,7 @@ stages:
               ADO_API_TOKEN: $(System.AccessToken)
               DANGER_GITHUB_API_TOKEN: $(githubPublicRepoSecret)
               TARGET_BRANCH_NAME: ${{ variables.targetBranchName }}
+              DEVTOOL_TELEMETRY_TOKEN: $(devtools-telemetry-key)
```

Comment:
This makes me think that the bundle size comparison actually fails if we don't provide some value?

Reply:
Yea. I wonder `undefined` environment variable will be a problem but removed it from here now since we have a local .env file